gridstatus/ercot.py

```diff
@@ -671,6 +682,115 @@ def _handle_load_forecast(self, doc, forecast_type, verbose=False):
 
         return df
 
+    def get_committed_capacity(self, date="latest", verbose=False):
+        """
+        Retrieves the actual committed capacity (the amount of power available from
+        generating units that were on-line or providing operating reserves).
+
+        Data is ephemeral and does not support past days.
+        """
+        data = self._get_capacity_dataset(verbose=verbose)
+
+        return (
+            data.loc[
+                # Actual values
+                data["forecast"] == 0,
+                ["Interval Start", "Interval End", "capacity"],
+            ]
+            .rename(columns={"capacity": "Capacity"})
+            .reset_index(drop=True)
+        )
+
+    def get_forecasted_capacity(self, date="latest", verbose=False):
```

Comment:
let's call this `get_capacity_forecast` more consistent with `get_load_forecast`, `get_solar_forecast`, etc